### DISCUSSION TITLE
On UK maps, only OSM "Trunk Roads" should have yellow-on-green road markers

### DISCUSSION THREAD

**Discussion by GCarty80** [organicmaps/organicmaps#1243] [Ideas]
Currently the route numbers of all A roads in the UK are displayed using yellow-on-green road markers, while those of B roads are displayed using black-on-white road markers.

This is not correct as not all A roads are classified as primary routes (which use the yellow-on-green markers in UK signage). I suspect the issue has arisen because normal UK parlance for road classification does not match up with OpenStreetMap's system.

Primary A-roads in the UK are almost invariably classified as "Trunk Roads" in OSM, and thus only OSM Trunk Roads should have the yellow-on-green markers -- OSM "Primary Roads" in the UK are actually _non-primary_ A roads and should have black-on-white road markers like those used for OSM "Secondary Roads" (UK B roads).

(Now an Issue - see https://github.com/organicmaps/organicmaps/issues/1244)

**Comment by biodranik** [comment]
Thanks! Do other OSM-based navigators display markers correctly?

**Reply by GCarty80** [reply]
I wouldn't know, as the only one I used before adopting Organic Maps was its predecessor MAPS.ME.

**Comment by biodranik** [comment]
Please create an issue with a more generic name, and provide some screenshots with examples. The idea is to motivate users from other countries to report issues with their road signs too.

**Reply by GCarty80** [reply]
How do I include screenshots on messages in this forum?

**Reply by biodranik** [reply]
Drag them into the text edit box, or use the "attach image" button from a mobile device.

**Reply by GCarty80** [reply]
As far as other countries are concerned, perhaps it would be nice if the Germans could have their "blue crushed hexagon inside white rectangle" autobahn marker, and the similar marker in red instead of blue for the Swiss?

**Reply by GCarty80** [reply]
OK, I've now created an issue at https://github.com/organicmaps/organicmaps/issues/1244

**Comment by GCarty80** [comment]
Incidentally, is this kind of info ("which route markers to use with each class of roads in a given country") something that's stored within the Organic Maps app code itself, or is it stored within the sets of map data that we download into the app?

**Reply by biodranik** [reply]
@vng should know

**Reply by vng** [reply]
Can't say for sure now. Will investigate this issue later ..
